src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgreSQL.java

```diff
@@ -88,12 +88,11 @@
     private volatile FileLock lock;
     private final boolean cleanDataDirectory;
 
-    EmbeddedPostgreSQL(File parentDirectory, File dataDirectory, boolean cleanDataDirectory, Map<String, String> postgresConfig) throws IOException
+    EmbeddedPostgreSQL(File parentDirectory, File dataDirectory, boolean cleanDataDirectory, Map<String, String> postgresConfig, int port) throws IOException
```

Comment:
The constructor is package private, the only publicly accessible way of creating it is via the builder below. That said I'm happy to add another constructor - it would only be a couple of lines and might leave things a little tidier.